todoapp/src/components/Header.vue

```diff
@@ -1,16 +1,71 @@
 <!-- eslint-disable vue/multi-word-component-names -->
 <script setup lang="ts">
-import DeleteTasks from './DeleteTasks.vue'
+import plus from '../assets/plus.svg'
+import type { TextIdString } from '../types/text'
+import { computed } from 'vue'
+const props = defineProps<{
+  todos: TextIdString[]
+  modelValue: string
+}>()
 
-const emit = defineEmits(['deleteTasks'])
+const emit = defineEmits<{
+  (e: 'addTodo', todo: string): void
+  (e: 'update:modelValue', value: any): void
+}>()
 
-function deleteTasks() {
-  emit('deleteTasks')
+const inputValue = computed({
```

Comment:
You don't need to have this inputValue in the header component (you just have a title and a button :))